lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -115,8 +115,19 @@ sub _add_routes {
         $r->name("$self->{route_prefix}$name") if $name;
       }
 
-      $self->_add_default_response($op_spec, $_) for @{$self->{default_response_codes}};
-
+      if ($self->validator->version == 2) {
+        $self->_add_default_response($op_spec, $_) for @{$self->{default_response_codes}};
+      } elsif ($self->validator->version == 3 and @{$self->{default_response_codes}}) {
+        my $root_spec = $self->validator->get(['/']);;
```

Comment:
Why not do `get('/components/responses')` instead? Anyways, I think there's way too much code here. Let's do something like this instead:

```
my $add_default_response_method = sprintf "_add_default_response_%s", $self->validator->version;
$self->$add_default_response_method($op_spec, $_) for @{$self->{default_response_codes}};
...
sub _add_default_response_2 {
}
sub _add_default_response_3 {
}
```